.github/workflows/tag_action.yml

```
@@ -3,25 +3,68 @@ name: Publish release
 on:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+**'
```

What is `**` for?

---

This will allow for longer versioning schemes so now it will allow for example "v1.2.3.4" the restricion is that it has to start with "v", and have 3 numbers separated by dot, next there can be anything

---

Based on [nrf history](https://github.com/nrfconnect/sdk-nrf/tags) I would rather expect -rcX, -devX instead of next number. 

```suggestion
      - 'v[0-9]+.[0-9]+.[0-9]+(-[a-z]+[0-9]+)*'
```